src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
is `el.getAttribute('tabIndex')` different from `el.tabIndex`? I thought we said the latter was preferably last time